otter/models/cass.py

```diff
@@ -808,7 +809,24 @@ def _do_update(ts, _):
                  'status': status.name},
                 DEFAULT_CONSISTENCY)
 
-        return self.view_config().addCallback(_do_update)
+        @self.with_timestamp
+        def set_deleting(ts, _):
```

Comment:
`view_config` is called to raise nogrouperror when `update_status` is called on non-existent group. However, I just realized that it will raise nogrouperror on DELETING group as well which is probably ok since setting any status on a deleting group is not helpful? What do you think?